viewer/src/utilities/metrics.ts

```diff
@@ -26,8 +26,13 @@ let globalLogMetrics = true;
 const globalProps = {
   VERSION,
   project: 'unknown',
-  application: 'unknown'
+  application: 'unknown',
+  // Use a random identifier because we want to don't track users over multiple sessions to not
+  // violate GDPR.
+  sessionId: generateUuidv4()
 };
+// Don't identify users in MixPanel to avoid GDPR problems
+const mixpanelDistinctId = 'reveal-single-user';
```

Comment:
```suggestion
  application: 'unknown',
};
```

I don't think we really need it. Mixpanel already assigns a random id to a user. https://help.mixpanel.com/hc/en-us/articles/360041039771-Getting-Started-with-Identity-Management#identification-examples